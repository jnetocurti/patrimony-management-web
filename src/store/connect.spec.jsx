import React from 'react';

import { Typography } from '@material-ui/core';

import { mount } from 'enzyme';

import Store from '.';
import Connect from './connect';

describe('src/store/connect', () => {
  it('should render correctly', () => {
    const ConnectableElement = Connect()(({ test }) => (
      <Typography>{test}</Typography>
    ));

    const component = mount(
      <Store>
        <ConnectableElement />
      </Store>
    );
    expect(component).toMatchSnapshot();
  });
});
