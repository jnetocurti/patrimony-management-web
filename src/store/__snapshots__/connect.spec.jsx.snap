// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/store/connect should render correctly 1`] = `
<Store>
  <Component>
    <Component
      dispatch={[Function]}
    >
      <WithStyles(ForwardRef(Typography))>
        <ForwardRef(Typography)
          classes={
            Object {
              "alignCenter": "MuiTypography-alignCenter",
              "alignJustify": "MuiTypography-alignJustify",
              "alignLeft": "MuiTypography-alignLeft",
              "alignRight": "MuiTypography-alignRight",
              "body1": "MuiTypography-body1",
              "body2": "MuiTypography-body2",
              "button": "MuiTypography-button",
              "caption": "MuiTypography-caption",
              "colorError": "MuiTypography-colorError",
              "colorInherit": "MuiTypography-colorInherit",
              "colorPrimary": "MuiTypography-colorPrimary",
              "colorSecondary": "MuiTypography-colorSecondary",
              "colorTextPrimary": "MuiTypography-colorTextPrimary",
              "colorTextSecondary": "MuiTypography-colorTextSecondary",
              "displayBlock": "MuiTypography-displayBlock",
              "displayInline": "MuiTypography-displayInline",
              "gutterBottom": "MuiTypography-gutterBottom",
              "h1": "MuiTypography-h1",
              "h2": "MuiTypography-h2",
              "h3": "MuiTypography-h3",
              "h4": "MuiTypography-h4",
              "h5": "MuiTypography-h5",
              "h6": "MuiTypography-h6",
              "noWrap": "MuiTypography-noWrap",
              "overline": "MuiTypography-overline",
              "paragraph": "MuiTypography-paragraph",
              "root": "MuiTypography-root",
              "srOnly": "MuiTypography-srOnly",
              "subtitle1": "MuiTypography-subtitle1",
              "subtitle2": "MuiTypography-subtitle2",
            }
          }
        >
          <p
            className="MuiTypography-root MuiTypography-body1"
          />
        </ForwardRef(Typography)>
      </WithStyles(ForwardRef(Typography))>
    </Component>
  </Component>
</Store>
`;
